Podcast/BookmarkTableViewCell.swift

```diff
@@ -163,6 +173,7 @@ class BookmarkTableViewCell: UITableViewCell {
     }
     
     func setPlayButtonToState(isPlaying: Bool) {
+        //TODO: change slider value + probably remove with key value observation
```

Comment:
No definitely want to leave the state stuff. This is what would be called by the setup method from KVO